hazelcast/src/test/java/com/hazelcast/config/YamlConfigImportVariableReplacementTest.java

```diff
@@ -587,8 +581,6 @@ public void testImportRedefinesSameConfigScalarThrows() throws Exception {
                 + "    - ${config.location}\n"
                 + "  cluster-name: name2";
 
-        rule.expect(new RootCauseMatcher(InvalidConfigurationException.class, "hazelcast/cluster-name"));
```

Comment:
I think we should keep this expectation here and below. Moving it to the `@Test` annotations removes the coverage for printing the YAML "path" led to the exception.